sql_export_mail/views/sql_export_view.xml

```diff
@@ -27,12 +27,7 @@
                         <field name="mail_user_ids" nolabel="1" />
                     </group>
                     <group string="Crons" groups="base.group_system">
-                        <field
-                            name="cron_ids"
-                            nolabel="1"
-                            domain="[('function', '=', '_run_all_sql_export_for_cron')]"
```

Comment:
Why remove the domain here?
I believe it is important, else, the user could choose just any cron, and it make no sense to choose a cron which is not related with the sql export mail feature. (It won't have any effect to choose these cron.
Since it may be difficult to add a domain on `code`, I guess a domain to restrict cron linked to `model_id` `sql.export`

Reply:
hey, @florian-dacosta thanks for the suggestion I have check it works properly and adds domain according to the standard framework. 
can you please review this pr?